### PR TITLE
Add links to serverless library

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
Fixes https://github.com/Microsoft/vscode-azurefunctions/issues/1047

Screen Shot 2019-04-22 at 10 25 04 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Create, debug, manage, and deploy Azure Functions directly from VS Code. Check
 out this [deployment tutorial](https://code.visualstudio.com/tutorials/functions-extension/getting-started)
-to get started with the Azure Functions extension.
+to get started with the Azure Functions extension and check out the [Azure serverless community library](https://aka.ms/AA4ul9b) to view sample projects.
```
Comment (nturinski) [inline README.md]: The [link](https://aka.ms/AA4ul9b) doesn't seem to be taking me anywhere.

Comment (nturinski) [reply README.md]: Never mind, it just took forever to load

File: src/commands/createNewProject/NewProjectLanguageStep.ts [hunk 2]
```diff
@@ -42,20 +43,29 @@ export class NewProjectLanguageStep extends AzureWizardPromptStep<IProjectWizard
...
 
+        languagePicks.push({ label: localize('viewSamples', '$(link-external) View sample projects'), data: undefined, suppressPersistence: true });
+
         const options: QuickPickOptions = { placeHolder: localize('selectFuncTemplate', 'Select a language for your function project') };
-        wizardContext.language = <ProjectLanguage>(await ext.ui.showQuickPick(languagePicks, options)).label;
+        const result: ProjectLanguage | undefined = (await ext.ui.showQuickPick(languagePicks, options)).data;
+        if (result === undefined) {
+            await openUrl('https://aka.ms/AA4ul9b');
+            wizardContext.actionContext.properties.cancelStep = 'viewSampleProjects';
+            throw new UserCancelledError();
```
Comment (nturinski) [inline src/commands/createNewProject/NewProjectLanguageStep.ts]: Didn't we have a custom error wrapper so we didn't have to use the `UserCancelledError`?

Comment (EricJizbaMSFT) [reply src/commands/createNewProject/NewProjectLanguageStep.ts]: Uh there's a few things we can do related to `actionContext`, but in this case I think a "cancel" makes sense. Basically they're "cancelling" creating the project in order to view sample projects instead

**Pull request merged**